packages/plugins/rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -19,31 +19,28 @@ Requires: git
 BuildRequires: git
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(diffy)
-Requires: %{?scl_prefix}rubygem(git)
+Requires: ruby
 BuildRequires: foreman-assets >= %{foreman_min_version}
 BuildRequires: foreman-plugin >= %{foreman_min_version}
-BuildRequires: %{?scl_prefix}rubygem(diffy)
-BuildRequires: %{?scl_prefix}rubygem(git)
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby
+BuildRequires: ruby
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 Provides: foreman-plugin-%{plugin_name} = %{version}
+BuildRequires: rubygem(diffy)
+BuildRequires: rubygem(git)
 # end specfile generated dependencies
 %{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
 
 # start package.json devDependencies BuildRequires
-BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
-BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 10.1.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 11.0.0
-BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) >= 3.0.0
-BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) < 4.0.0
+BuildRequires: npm(@babel/core) >= 7.7.0
+BuildRequires: npm(@babel/core) < 8.0.0
+BuildRequires: npm(@theforeman/builder) >= 10.1.0
+BuildRequires: npm(@theforeman/builder) < 11.0.0
+BuildRequires: npm(@theforeman/stories) >= 10.1.0
+BuildRequires: npm(@theforeman/stories) < 11.0.0
```

Comment:
This breaks things for some reason:
```
DEBUG util.py:634:  No matching package to install: 'npm(@theforeman/builder) < 11.0.0'
DEBUG util.py:634:  No matching package to install: 'npm(@theforeman/stories) >= 10.1.0'
DEBUG util.py:634:  No matching package to install: 'npm(@theforeman/stories) < 11.0.0'
```

Also, I wonder about stories. We dropped that in Foreman itself.